src/appleseed.studio/python/module.cpp

```diff
@@ -68,7 +68,7 @@ Project* new_project()
 
 Project* open_project(const char* project_path)
```

Comment:
I prefer the boolean. load_project if it fails it returns false and leaves the previously opened project (if any) open. Users can't test studio.current_project() because that might point to another project.

Also, I don't think there's a requirement for open_project to return a Project.
The real purpouse of the function is to replace the currently opened project in studio by a new one. We should not promote users storing references to studio Projects in variables. 
They will become invalid the next time you do new or open project.

Lets return a boolean from open project. If it's possible, lets try to print the open error (if any) to the python console to give some feedback to users, and for consistency, lets not return anything from new_project.